### PR TITLE
Add srm

### PR DESCRIPTION
**Pull request by jakirkham**
`srm` is basically like `rm`, but does secure deletion by performing multiple overwrites of a file before deleting it.

### CODE REVIEW

**Comment by jakirkham** [comment]
Thoughts @conda-forge/staged-recipes?

File: recipes/srm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  url: http://download.sourceforge.net/srm/srm-{{ version }}.tar.gz
+  sha256: 7583c1120e911e292f22b4a1d949b32c23518038afd966d527dae87c61565283
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
```
Comment (CJ-Wright) [inline recipes/srm/meta.yaml]: Does this need make?

Comment (jakirkham) [reply recipes/srm/meta.yaml]: Hmm...we've been using the system copy for a while. Are we changing that?

Comment (CJ-Wright) [reply recipes/srm/meta.yaml]: I think so, especially at the lower level. CC @mariusvniekerk who was doing some work on these things yesterday.

Comment (scopatz) [reply recipes/srm/meta.yaml]: I think we don't have a consistent policy on this, so it has pretty much been up to the library author

Comment (jakirkham) [reply recipes/srm/meta.yaml]: In that case I'm inclined to leave it out unless people feel strongly about it.

Comment (scopatz) [reply recipes/srm/meta.yaml]: OK!

**Pull request merged**